add hint for day one compatibility of plugins to angular 19 announcement (#3439)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-0-0-angular-19-upgrade-and-standalone-components.md

As part of this change, the pipe, directives and components that are offered as part of the `@c8y/ngx-components` library, will now be available in standalone mode.

To ensure a smooth transition to Angular 19 and help you to adapt your implementations, we will provide an upgrade documentation when this change happens. This documentation will help you navigate through the changes and update your codebase accordingly.

> In order to ensure a day one compatibility of your plugins with the applications based on the v1022 Web SDK, you can already today add the standalone flag to all your components/pipes/directives of your plugin.

Reviewer: ```suggestion
To ensure a day one compatibility of your plugins with applications based on the v1022 Web SDK, you should already today add the standalone flag to all components/pipes/directives of your plugins.
```
